strongbox-data-service/pom.xml

```diff
@@ -78,11 +78,6 @@
             <artifactId>strongbox-db-server</artifactId>
             <version>1.0-SNAPSHOT</version>
         </dependency>
-        <dependency>
-            <groupId>${project.groupId}</groupId>
-            <artifactId>strongbox-db-liquibase</artifactId>
```

Comment:
@erikweibust ,

Have you actually tried building the code without this dependency? Doesn't this module fail to build?

Reply:
Yeah.  First thing I did was run a "mvn clean install". I just ran it again.

[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 01:14 min
[INFO] Finished at: 2019-10-03T07:07:13-05:00
[INFO] ------------------------------------------------------------------------

Reply:
Great! Thanks for confirming! :)